pkg/repo2cpe/testdata/repository-to-cpe.json

```diff
@@ -2,23 +2,37 @@
   "data": {
     "3scale-amp-2-rpms-for-rhel-8-x86_64-debug-rpms": {
       "cpes": [
+        "cpe:/a:redhat:3scale:2.13::el8",
         "cpe:/a:redhat:3scale_amp:2.10::el8",
         "cpe:/a:redhat:3scale_amp:2.11::el8",
+        "cpe:/a:redhat:3scale_amp:2.12::el8",
         "cpe:/a:redhat:3scale_amp:2.8::el8",
         "cpe:/a:redhat:3scale_amp:2.9::el8"
+      ],
+      "repo_relative_urls": [
+        "content/dist/layered/rhel8/x86_64/3scale-amp/2/debug"
       ]
     },
     "3scale-amp-2-rpms-for-rhel-8-x86_64-rpms": {
       "cpes": [
+        "cpe:/a:redhat:3scale:2.13::el8",
```

Comment:
@RTann Why did you add these additional CPEs to test the new `repo_relative_urls` field?

Reply:
I just copied it directly from the current version of https://access.redhat.com/security/data/metrics/repository-to-cpe.json. Yes, it does not affect the test and added some noise, but I figured I might as well just copy over the entire current state of these entries

Reply:
:+1: